src/systemjs-webpack-interop.d.ts

```diff
@@ -6,7 +6,9 @@ declare module "systemjs-webpack-interop" {
   export function modifyWebpackConfig(
     webpackConfig: WebpackConfig
   ): WebpackConfig;
-  export function verifyWebpackConfig(webpackConfig): void;
+  export function verifyWebpackConfig(
+    webpackConfig: WebpackConfig
```

Comment:
Does this require that we depend on `@types/webpack`?

Reply:
Or actually it looks like we're already using WebpackConfig on line 7. Looks like that is defined by the `webpack` library, without the need to install `@types/webpack`. So no need to change this 😄

Reply:
I’m using your plugin in .ts files. I just write `setPublicPath(...)` but it prompts the error below:

```
9:39 Parameter 'webpackConfig' implicitly has an 'any' type.
     7 |     webpackConfig: WebpackConfig
     8 |   ): WebpackConfig;
  >  9 |   export function verifyWebpackConfig(webpackConfig): void;
       |                                       ^
    10 | 
    11 |   type WebpackConfig = object | WebpackConfigGetter;
    12 |   interface WebpackConfigGetter {
```